lib/ini.js

```diff
@@ -9,7 +9,7 @@ const encode = (obj, opt = {}) => {
   opt.sort = opt.sort === true
   opt.whitespace = opt.whitespace === true || opt.align === true
   /* istanbul ignore next */
-  opt.platform = opt.platform || process?.platform
+  opt.platform = opt.platform || (typeof process !== 'undefined' && process.platform)
```

Comment:
```suggestion
  opt.platform = opt.platform || (process && process.platform)
```

A truthiness check is sufficient here.